recipes/cgal/all/conandata.yml

```diff
@@ -17,6 +17,15 @@ sources:
   "5.1.1":
     sha256: eb052f9b47e4f5891f217b667f34e0e3aee9ff36b7cf858cd771d555ae9c5c30
     url: https://github.com/CGAL/cgal/releases/download/v5.1.1/CGAL-5.1.1.zip
+  "5.1.2":
+    sha256: e499ad8199580fa979a5a6e431e667c22ca2892ebbb7981e699849060645341d
+    url: https://github.com/CGAL/cgal/releases/download/v5.1.2/CGAL-5.1.2.tar.xz
+  "5.1.3":
+    sha256: 6986e6b67ec3fe4191f35e326bb2838995b84090448a15a8050347c4dcab7829
+    url: https://github.com/CGAL/cgal/releases/download/v5.1.3/CGAL-5.1.3.tar.xz
```

Comment:
do you need both?

Reply:
do you mean removing 5.1.1 and 5.1.2?

Reply:
We typically discourage adding older version since they are never used. Unless you specifically need an exact one...

```suggestion
```

Reply:
So I also remove `5.0.1` and `5.0.2`?

Reply:
No, we can not remove versions that already have been merged

Reply:
You can't remove versions already added before your PR. The question is: do you really need 5.1.2 while you also add 5.1.3?